codec-fix/README.md

```diff
@@ -18,6 +18,11 @@ fieldConverterClassName: com.exactpro.sf.services.fix.FixFieldConverter
 removeTrailingZeros: false
 ```
 
+By default, FIX codec implementation expects that tags will be separated by _SOH_ (like a regular FIX message). However, if messages you want to decode/encode have different field separator you can set the custom separator using the **fieldSeparator** parameter. Example,
+```yaml
+fieldSeparator: '|'
+```
+
 #### decodeByDictionary
 
 The dictionary will be used to decode messages. Otherwise, messages will be decoded as flatten structure - no header, trailer or any components and groups
```

Comment:
The dictionary will be used to decode messages. Otherwise, messages will be decoded as **a** flatten structure - **without** header, trailer or any components and groups